talk/owt/sdk/include/cpp/owt/base/globalconfiguration.h

```diff
@@ -88,6 +88,31 @@ class GlobalConfiguration {
     hardware_acceleration_enabled_ = enabled;
   }
 #endif
+
+#if defined(WEBRTC_WIN)
```

Comment:
IIUC, SR can be configured per renderer, why it's a global configuration?

Reply:
Current windows renderer design, application can only pass an HWND handle to SDK from application. It is not able to configure the internal renderer impl.

If user wants per-renderer configure, they can attach their external renderer provided by application and do customization there.